webpack.config.js

```diff
@@ -7,6 +8,9 @@ module.exports = {
     path: path.join(__dirname, '/client/dist'),
     filename: 'bundle.js'
   },
+  plugins: [
+    new Dotenv()
```

Comment:
created a new instance of Dotenv for the app, maybe something kinda like express.Router()?

Reply:
Not like the Router. This is a webpack plugin, so when webpack compiles our React code into proper ES5 javascript (in our case, via npm run client-dev), it will also use this plugin in that process.